site/src/components/form.md

```diff
@@ -188,21 +189,100 @@ If you want to skip the basic validation on an input, apply a `data-skip-validat
 		</div>
 	</div>
 	<div class="field">
-		<label class="field-label" for="sample-input">
+		<label class="field-label" for="sample-skip-input">
 			Input
 			<span class="required-indicator"></span>
 		</label>
 		<div class="field-body">
 			<input
-				id="skip-input"
-				name="skip-input"
+				id="sample-skip-input"
+				name="sample-skip-input"
 				class="input"
 				type="text"
 				value=""
+				required
 				data-skip-validation
 			/>
 		</div>
 	</div>
+	<div class="field">
```

Comment:
Perhaps a separate PR, but I wanted to point out that there will be more work required in order to validate a group of checkboxes with form behavior. Right now, it only accounts for one checkbox and the error message is using the checkbox text.

![image](https://user-images.githubusercontent.com/94572161/225480200-9116a077-773c-4108-a936-656405e14e26.png)


![image](https://user-images.githubusercontent.com/94572161/225479460-92ba07a5-dcf5-46d1-adcc-5130a46931a5.png)

Reply:
I didn't mean to cover group selection at the moment.